website/source/docs/providers/azurerm/index.html.markdown

```diff
@@ -115,7 +115,24 @@ To grant permissions to the App Registration to your subscription, you now must
 - Select **Subscriptions** from the left panel. Select the subscription that you want to use. In the Subscription details pane, click **Access Control (IAM)**
 - Click **Add**.  For Step 1 select an appropriate role for the tasks you want to complete with Terraform. You can find details on the built in roles [here](https://azure.microsoft.com/en-gb/documentation/articles/role-based-access-built-in-roles/)
 - Type in the name of the application added in the search box. You need to type this as it won't be shown in the user list. Click on the appropriate user in the list and then click **Select**
-- Click **OK** in the **Add Access** panel. The changes will now be saved   
+- Click **OK** in the **Add Access** panel. The changes will now be saved
+
+To create using azure cli:
+```
+az login
```

Comment:
not sure I based it on the doc I found...I need to validate...I am hoping this is not another command/utility they are using.